pubspec.lock

```diff
@@ -7,21 +7,21 @@ packages:
       name: _fe_analyzer_shared
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "61.0.0"
+    version: "54.0.0"
```

Comment:
dependabot pub does not effect the pubspec.yaml directly, and when it does bump versions, it doesnt care about bumping past the `sdk:` field declared in the pubspec.yaml

For example: analyzer `5.13.0` requires dart `2.19`, but dependabot happily bumped the lockfile to an invalid version. [this](https://github.com/Workiva/scip-dart/pull/88/files#diff-dd4fbda47e51f1e35defb9275a9cd9c212ecde0b870cba89ddaaae65c5f3cd28R4) change is an effort to see what a different versioning strategy does to the resolution logic in dart